modules/pensions/spec/controllers/v0/claims_controller_spec.rb

```diff
@@ -31,7 +31,7 @@
       expect(monitor).to receive(:track_create_attempt).once
       expect(monitor).to receive(:track_create_validation_error).once
       expect(monitor).to receive(:track_create_error).once
-      expect(claim).not_to receive(:upload_to_lighthouse)
+      expect(controller).not_to receive(:process_and_upload_to_lighthouse)
```

Comment:
where is `controller`? i am surprised you did not get a lint error about mocking the 'subject under test', so instead check that the job.perform_async does not get called?

Reply:
`controller` is an automatic var in controller specs, I feel like in the past I've tried to use it and it would throw a listing error.